bin/react-to-product-owners-yml-changes.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
Would it be possible to add a test for this binary, even just one, just to give an idea of the expected usage? We could create a mock product-owners file and test against that. Otherwise this code interacts with some far off (closed source) repo in a way that's pretty difficult to deduce from the code alone.

Reply:
Looks like https://github.com/getsentry/security-as-code/pull/311 has already gone in. If your concern is expected usage, let me generate the expected file here to clear up the ambiguity. I don't think there's an easy way to test just this and don't think it's super helpful to test in CI, unless our team is making changes to the file.

Reply:
added!

Reply:
Thanks! Though see my comment on product-owners.yml, maybe we should have mock that's clearly a test file, rather than the actual data?

Reply:
We would want the actual data to be here, and it's not sensitive so it's fine.